tests/bin/test-integration-ec2.sh

```diff
@@ -0,0 +1,158 @@
+#!/bin/bash
+#
+# Preps a test environment and runs `make test-integration`
+# against artifacts produced from the current source tree
+#
+
+# fail on any command exiting non-zero
+# set -eo pipefail
+set -eo
```

Comment:
drat. We should remove the `-o` flag then since that's used to specify option flags.